frappe/widgets/form/meta.py

```diff
@@ -68,6 +68,7 @@ def _get_path(fname):
 			self.set("__listview_template", get_html_format(listview_template))
 
 		self.add_code_via_hook("doctype_js", "__js")
+		self.add_code_via_hook("doctype_list_js", "__js")
```

Comment:
shouldn't this be: `self.add_code_via_hook("doctype_list_js", "__list_js")` as per line number 63